app/services/rules_cleanup_service.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# This class cleans up unused rules and reimports
+class RulesCleanupService
+  class << self
+    def run!
+      # Clean up rules that don't belong to any profile
+      Rule.where.not(id: ProfileRule.select(:rule_id)).delete_all
+
+      # Clean up rules that belong only to canonical profiles
+      delete_rules(select_rules)
+
+      # Reimport all rules if we're not in the testing environment
+      Rake::Task['ssg:import_rhel_supported'].execute unless Rails.env.test?
```

Comment:
I'm not so sure about this, we already have a job that imports the SSGs and on a fresh install this would make it obsolete :thinking:

Reply:
I'd remove it from the migration. This should be a non-impactful cleanup.

Reply:
Okay, but somehow we should trigger a reimport in prod :thinking: